fabric-enchantments-v1/src/main/java/net/fabricmc/fabric/api/enchantment/FabricEnchantmentTarget.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.enchantment;
+
+import net.minecraft.item.Item;
+
+/**
+ * Interface for an enchantment target
```

Comment:
This should be more descriptive.
```suggestion
 * Allows mods to define custom enchantment targets outside of Vanilla's {@link EnchantmentTarget} enum.
 * Enchantment targets restrict what items a certain {@link FabricEnchantment} can be applied to.
```